04_python_tutorial/fork-and-clone.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+MAVEN_REPO="jenkins-docs/simple-java-maven-app"
```

Comment:
I wonder why we use Maven instead of Python repo here.

https://github.com/jenkins-docs/simple-python-pyinstaller-app

Reply:
I've corrected this